catkit/catkit_types.py

```diff
@@ -52,7 +52,29 @@ class ImageCentering(Enum):
 
 # Create shortcuts for using Pint globally.
 units = UnitRegistry()
-quantity = units.Quantity
+
+
+class Quantity(units.Quantity):
+    """ Wrapper for pint.Quantity to avoid compounding quantities.
+
+        https://pint.readthedocs.io/en/stable/developers_reference.html#pint.Quantity
+    """
+    # Add this alias due to __news__() shadowing of the above global units.
+    _units = units
+
+    def __new__(cls, value, units=None):
+        if isinstance(value, cls._units.Quantity):
```

Comment:
The base issue is that ``pint`` quantities are dynamically defined belonging to an instance of ``pint.UnitRegistry()``. This results in ``if isinstance(value, cls._units.Quantity)`` _always_ returning ``False`` on Windows due to the multiprocess spawn.

Updating ``_units = units`` with that on the spawn works only so far and then runs into the issue that ``pint`` quantities from diff registries aren't compatible with one another. We could just nuke an old instance and create a new but things are starting to become a slippery rabbit hole.

Another solution would be to not use ``pint`` from the parent process which includes it's use from scripts and class instantiations (From scripts), class/func defaults and other "compile"/import time usages. This would keep all ``quantity`` instances to a single registry and also from that of the spawned child process.